packages/dynamite/dynamite_end_to_end_test/test/nested_ofs_test.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:dynamite_end_to_end_test/nested_ofs.openapi.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('BaseNestedAllOf', () {
```

Comment:
Why not test oneOf and anyOf as well?

Reply:
Because due to our implementation of oneOf/anyOf we wouldn't test anything meaningful.
Both types just cache the data they got into a `JsonObject` and when serialized they just return that data.

Improving this to automatically use the first present value is already on my list (I should open an issue about that).

Reply:
I added them nevertheless but I don't think they really test any logic.